README.md

```diff
@@ -37,7 +37,17 @@ Head over to the [webapp](https://dhwaneetbhatt.com/strapd/) for an interactive
 
 ## CLI Installation
 
-### Quick Install (Recommended)
+### Homebrew (macOS/Linux)
+
+```bash
+# From the main repository (coming soon - pending tap setup)
```

Comment:
For now, let us only keep the `--build-from-source` option until I setup the homebrew tap. Then we can include this method.